<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>累死雞記帳</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="ExpenseTab" name="epenseTab">
      <attribute name="title">
       <string>支出</string>
      </attribute>
     </widget>
     <widget class="StatisticsTab" name="statisticsTab">
      <attribute name="title">
       <string>統計</string>
      </attribute>
     </widget>
     <widget class="SettingTab" name="settingTab">
      <attribute name="title">
       <string>設定</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StatisticsTab</class>
   <extends>QWidget</extends>
   <header>statisticstab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ExpenseTab</class>
   <extends>QWidget</extends>
   <header>expensetab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingTab</class>
   <extends>QWidget</extends>
   <header>settingtab.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
